ide/coqide/coqide.ml

```diff
@@ -1103,6 +1118,8 @@ let build_ui () =
         radio "Set diff" 1 ~label:"Show diffs: only _added";
         radio "Set removed diff" 2 ~label:"Show diffs: added and _removed";
       ];
+    item "Show Proof Diffs" ~label:"_Show Proof Diffs" ~accel:"<CTRL>S"
```

Comment:
isn't ^S already taken by File>Save?

Reply:
I'll make it Alt-Shift-S.  Would be nice if the key mapping was user-settable.